recipes/festival/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "festival" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/festival/archive/5db6ee949f4e4e9da25c87bd03b90e69f9011393.tar.gz
+  sha256: 9f0d0e4f0e91fca4430ee4a6ca4d3d24bac444f36b4f4e9653c6119aadfae13e
+
+build:
+  run_exports:
+    - {{ pin_subpackage('festival', max_pin='x.x') }}
```

Comment:
Because you have multiple outputs, I believe that you need to move this run_export into both of the output sections. Or at least add it to the libfestival-dev package. Please check the detemplated recipes in your built packages metadata.

Reply:
@carterbox I don't understand your request here. The run_exports are both in the global scope and in each individual package. Where else should I add it?

Reply:
Yep. Looks good. Maybe I was looking at some old commits.